contrib/helm/harbor/values.yaml

```diff
@@ -28,89 +28,41 @@
 persistence:
   enabled: false
 
-externalDomain: harbor.192.168.99.100.xip.io
-## tls_crt, tls_key, ca_crt should match the domain above
+# The tag for Harbor docker images
+harborImageTag: &harbor_image_tag v1.4.0
```

Comment:
I don't like having a common tag like this, especially if it can't be used for _all_  ( see `tag: v2.6.2-v1.4.0` ).  Also eventually we should look to break out each component into its own helm chart which would then require each service to have its own tag anyways.

Reply:
There is not a nice approach to support generating tag 'v2.6.2-v1.4.0'. Maybe we can specific a base tag 'v2.6.2' and concat('v2.6.2', '-', .Values.harborImageTag).
Will keep current design for now.